### PR TITLE
Global: Added acronym to manual project input

### LINKED ISSUE #161: Add the possibility to choose an acronym while creating a project manually
**Changes**
When adding a project manually, it should be possible to choose an acronym.

**Additional notes**
The change would prevent this from going unnoticed in a real document:
image

**Acceptance Criteria**
Extra input field where a project is added manually.

### PR DESCRIPTION
**Pull request by GeoffreyKarnbach**
## Description

Feature

#### What does this PR do?

Added an input field between the title and date inputs on the manual project component.

#### Dependencies

This Pull Request is strongly correlated to the backend pendant for it:
https://github.com/tuwien-csd/damap-backend/pull/241

### Checks

- [ ] Summary updated
- [ ] Version view updated
- [ ] Documentation added
- [ ] Tests added
- [ ] E2e tests created
- [ ] Successfully ran e2e tests before merge

closes GH-161

### CODE REVIEW

**Review by rekt-hard — COMMENTED** [review]
Functionality is there.

About the display: on a smaller screen size, the date input gets too small and the text is not readable.
image

**Comment by GeoffreyKarnbach** [comment]
Added a mediaquery, below a certain display width, the title, acronym and date selection all have the same size to ensure the readability of all elements.

In Fullscreen, the title still uses up about 60% of the space.

File: libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ export class ManualProjectInputComponent implements OnChanges {
     description: new UntypedFormControl(''),
     start: new UntypedFormControl(null),
     end: new UntypedFormControl(null),
+    acronym: new UntypedFormControl('', [Validators.maxLength(25)]),
```
Comment (rekt-hard) [inline libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.ts]: We do not have to be so restrictive here. Limiting it to 255 is enough.
Although it is called acronym, it is often also just a shorter name which could exceed 25 characters :)

Comment (GeoffreyKarnbach) [reply libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.ts]: Done

**Review by rekt-hard — APPROVED** [review]

**Pull request merged**